zerver/views/streams.py

```diff
@@ -857,7 +857,7 @@ def delete_in_topic(
     stream_id: int = REQ(converter=to_non_negative_int, path_only=True),
     topic_name: str = REQ("topic_name"),
 ) -> HttpResponse:
-    (stream, sub) = access_stream_by_id(user_profile, stream_id)
+    stream, _ = access_stream_by_id(user_profile, stream_id)
```

Comment:
The variable is unused either way; this just calls it `_`. We generally don't use `_` for this purpose, despite that being a common Python pattern, because our Django code uses `from django.utils.translation import gettext as _`, so doing this will  break translation code in this function.

Better to use `ignored_sub` if we want to indicate it's ignored, or doing `[0]`, though I think the `[0]` option feels a bit less readable.

Reply:
Hmm, I've changed this to `ignored_sub` but we have quite a bit of this `_` pattern through the codebase, which i think my intuition was to do it that that - if you e.g. `git grep ", _ ="`. Should we be removing all those and using a different pattern?